websites/F/Floatplane/presence.ts

```diff
@@ -0,0 +1,221 @@
+const presence = new Presence({
+    clientId: "889918462477095012"
+  }),
+  browsingTimestamp = new Date().getTime();
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+      largeImageKey: "logo",
+      startTimestamp: browsingTimestamp
+    },
+    pathnames = location.pathname.split("/");
+
+  if (pathnames[1] === "") {
```

Comment:
Probably a better check
```suggestion
  if (pathnames[1].length >= 0) {
```